build/nogo_config.json

```diff
@@ -253,6 +253,7 @@
       "planner/util": "planner code",
       "planner/implementation": "planner code",
       "planner/cascades": "planner code",
+      "planner/core/plan_cache.go": "planner code",
```

Comment:
Should we remove this before merging this PR?

Reply:
No, let's not make this file break again.